### PR TITLE
[sample_vpp] Add conversion support for Y210 and Y410 formats

### PR DESCRIPTION
**Pull request by egorovdanil**
Issue #1995

Signed-off-by: egorovdanil <danil.egorov@intel.com>

### CODE REVIEW

File: samples/sample_vpp/src/sample_vpp_parser.cpp [hunk 1]
```diff
@@ -414,6 +414,16 @@ mfxU32 Str2FourCC( msdk_char* strInput )
...
+#if (MFX_VERSION >= 1027)
+    else if ( 0 == msdk_stricmp(strInput, MSDK_STRING("y210")) )
+    {
+        fourcc = MFX_FOURCC_Y210;
+    }
+    else if ( 0 == msdk_stricmp(strInput, MSDK_STRING("y410")) )
+    {
+        fourcc = MFX_FOURCC_Y410;
+    }
+#endif
```
Comment (dmitryermilov) [inline samples/sample_vpp/src/sample_vpp_parser.cpp]: could you add else block and throw an exception if we can't find a corresponding if block

Comment (egorovdanil) [reply samples/sample_vpp/src/sample_vpp_parser.cpp]: @dmitryermilov, I did this via call vppPrintHelp() and return MFX_ERR_UNSUPPORTED status.

**Comment by dmitryermilov** [comment]
Thanks. LGTM. One last question: did you try it on TGL? I mean does NV12-> Y210 start working?

**Comment by egorovdanil** [comment]
> Thanks. LGTM. One last question: did you try it on TGL? I mean does NV12-> Y210 start working?

@dmitryermilov, I checked that yuy2 -> Y210 and yuy2 -> Y410 works on TGL.

**Comment by VUNNYSODHI** [comment]
Thanks @egorovdanil and @dmitryermilov for quick fix for this issue.

**Pull request merged**